content/20.treatments.md

```diff
@@ -206,139 +207,161 @@ In one study, CQ inhibited viral entry of SARS-CoV into Vero E6 cells, a cell li
+However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of antivirals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83, although the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be underpowered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
+Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC-alone group (mean of 0.062 versus 0.008; p=0.57).
+Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response, which could help to prevent multiorgan failure and death.
+However, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC.
 The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs. 0% in the SOC group, p=0.004).
 
-The study had several limitations.
-One of the limitations of the study was that the cohort mostly consisted of patients with mild to moderate symptoms, and the average age of analyzed patients was 46.
-This average may not be representative for COVID-19 patients because older age groups are known to be at higher risk.
-Although the authors claimed that HCQ could have beneficial effects in hindering disease progression, multiorgan failure, and death in COVID-19, this finding cannot be extrapolated to older patients or severe cases based on the evidence they present.
-Indeed, a larger sample needs to be tested to validate whether lymphocyte counts do increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.
-Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which appeared to introduce confounding effects.
-Thus, to better assess the effects of HCQ, an alternate study design needs to be outlined where SOC does not involve the use of anti-virals.
-In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., [@doi:10.1001/jama.2020.3786]); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.
+Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.
```

Comment:
The adverse effects of administration of HCQ to mild and moderate COVID-19 cases seemed manageable.